common-service-core/src/main/java/org/zowe/apiml/security/HttpsFactory.java

```diff
@@ -76,6 +77,10 @@ public ConnectionSocketFactory createSslSocketFactory() {
         }
     }
 
+    KeyStore getUsedStore() {
```

Comment:
This method has default access modifier, but what about to be clear put here comment to known:
- method is only for testing purpose
- KeyStore is stored only in case of empty keystore (not if keystore is provided)

Reply:
Good idea! I will add the comment.

Reply:
Makes sense. I have added the comment.